contracts/feature-tests/abi-tester/abi_tester_expected.abi.json

```diff
@@ -49,6 +49,7 @@
                 "Example endpoint docs."
             ],
             "name": "echo_abi_test_type",
+            "mutability": "Mutable",
```

Comment:
can we make the values all lowercase?
I'm guessing this should be easy using serde annotations in EndpointMutabilityAbiJson